gramps/plugins/webreport/repository.py

```diff
@@ -170,7 +170,7 @@ def repositorylistpage(self, report, the_lang, the_title, repos_dict, keys):
                     Html("th", "&nbsp;", class_="ColumnRowLabel", inline=True),
                     Html("th", self._("Type"), class_="ColumnType",
                          inline=True),
-                    Html("th", self._("Name", "Repository "),
```

Comment:
Yes.  There is also a similar trailing space in the `gramps/plugins/webreport/media.py` file.

Reply:
This word insert to html code, where all extra whitespaces are deleted when display.
And we see one space between words all time.